src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -75,6 +84,12 @@ export class FilterTimestamp extends HTMLElement {
     }
   }
 
+  setTimestampStatus() {
+    if (this.timeStamp != null || Object.keys(this.timeStamp).length !== 0) {
+      this.classList.add('expanded');
```

Comment:
This should go through the dropdowncontroller, shouldn't it?

When I set a time range, the dropdown appears.
When I open another dropdown or click anywhere else, the timestamp doesn't disappear.

Reply:
Sorry, Yes, it is. I just saw it after pushing the last commit f275fef. Thanks for noticing.